modules/deployment-scripts/create-kv-certificate/create-kv-cert.sh

```diff
@@ -14,7 +14,7 @@ do
         policy=$(az keyvault certificate get-default-policy \
             | sed -e s/CN=CLIGetDefaultPolicy/CN=${certCommonName}/g )
```

Comment:
📝 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)